scripts/lint/ourlint.js

```diff
@@ -133,7 +133,9 @@ ${invalids.map((icon) => `${format(icon)} ${findPositon(expectedOrder, icon)}`).
 		 * @returns {boolean} Whether the URL is raw GitHub asset URL.
 		 */
 		const isRawGithubAssetUrl = ($url) =>
-			$url.hostname === 'raw.githubusercontent.com';
+			$url.hostname === 'raw.githubusercontent.com' ||
+			($url.hostname === 'gihtub.com' &&
```

Comment:
Oopsie! Caught a typo there, I think :)
Pretty sure it should be "github" not "gihtub", right?